ui/test/surface_value_kind_test.cpp

```
@@ -0,0 +1,235 @@
+
+using namespace forge::ui;
+using forge::uitest::Harness;
+
+namespace {
+
+// EVERY kind, listed once. This vector is the test's own transcription of the
+// enum and it exists so a kind added without a name shows up as a duplicate or an
+// empty string below rather than as nothing at all.
+const std::vector<IrValueKind>& allKinds() {
+  static const std::vector<IrValueKind> kinds = {IrValueKind::None, IrValueKind::Profile,
+                                                 IrValueKind::Wire, IrValueKind::Solid,
```

Cppcheck (2.21.0)

[style] The function 'checkLeInt' is never used.

(unusedFunction)